### ISSUE TITLE
Stopped receiving notifications and can't add new webhooks.

### ISSUE THREAD

**Issue by braincomb** [IanMitchell/sentrydiscord.dev#44]
Stopped receiving Discord alerts since February 29, and can't seem to add any new webhooks on the website. Always getting `Sorry, something went wrong. Please try again later.`

**Comment by IanMitchell** [comment]
It appears as if the database was deleted 😬. I'll see what I can do but this may not be recoverable unfortunately

**Comment by Bukk94** [comment]
That's unfortunate, but I guess not the end of the world. We can always setup new link. Do you think you can get database up and running so we can create new webhooks?

**Comment by IanMitchell** [comment]
Yeah, gonna work on that today and tomorrow. Out of curiosity, have you all tried the new official Discord integration and bot from Sentry? Does this service still provide any value compared to that?

**Comment by braincomb** [comment]
> Yeah, gonna work on that today and tomorrow. Out of curiosity, have you all tried the new official Discord integration and bot from Sentry? Does this service still provide any value compared to that?

Thanks @IanMitchell , keep us updated!

As far as the official integration goes, I believe that it's not available on a Free plan, so I  was not able to try it. So this service definitely still provides value!

**Comment by IanMitchell** [comment]
Ah, so it is :( that is too bad. I'll post an update here once I get it up and running again!

**Comment by Mendes11** [comment]
> Yeah, gonna work on that today and tomorrow. Out of curiosity, have you all tried the new official Discord integration and bot from Sentry? Does this service still provide any value compared to that?

It lacks too much compared with this tool.  It won't show the stacktrace, you can only add tags statically, and there isn't an option to include all tags in the message. The only pro of using that one is that they have buttons...

![image](https://github.com/IanMitchell/sentrydiscord.dev/assets/7904704/678eb8d9-04e5-4968-895d-4b9ba93a062a)

**Comment by IanMitchell** [comment]
The website should be back online and accessible. Sorry for the inconvenience all 😞, really appreciate your patience with me getting it set up again

**Comment by IanMitchell** [comment]
Gonna edit the title + pin this post for others to find if needed. Thanks for the helpful reports + information on the official plugin all, super helpful.

**Comment by Bukk94** [comment]
Thank you so much for your effort! I can confirm the service is up and running again :) Just generated a new URL.
